src/content/docs/infrastructure/host-integrations/host-integrations-list/jmx-monitoring-activemq.mdx

```diff
@@ -0,0 +1,143 @@
+---
+title: JMX monitoring with ActiveMQ
+tags:
+  - Integrations
+  - On-host integrations
+  - On-host integrations list
+  - ActiveMQ dashboard
+  - ActiveMQ with New Relic
+metaDescription: 'Get a tour of our quickstarts and dashboards that take JMX-scraped data from your favorite automation software and houses it in the New Relic platform.'
+--- 
+
+import activemqLandingPage from 'images/activemqLandingPage.png'
+
+import accessDashboards from 'images/Access_dashboards.png' 
+
```

Comment:
think we just have an 'infrastructure agent', not different infrastructure agents

Reply:
Good catch! removed!